keycloak/keycloak-admin-ui
# Realm settings(keys): toggle provider modal for all provider types

## Motivation


## Brief Description


## Verification Steps



1. Go to Realm settings -> Keys -> Providers tab.
2. Create a provider for each of the options listed.
3. Verify that the provider shows in the table.

## Checklist:


## Additional Notes
